### DISCUSSION TITLE
Test Scripts

### DISCUSSION THREAD

**Discussion by nmskhan** [CoBrALab/RABIES#220] [Q&A]
Hi All
Thank you all for putting together awesome scripts and tools for functional image processing. Do you anyone happen to have a few test scripts to run on the example dataset to see how this works? I noticed that the example data sets were collected in the coronal (horizontal) plane, wondering if the data has to be in the cor plane or any conflict during coregistration with the atlas?
Please shed some light on these.
Thanks
Ned

**Comment by Gab-D-G** [comment]
Hi Ned, I'm glad to hear about your interest! 

I would recommend following the example commands shown in https://rabies.readthedocs.io/en/stable/running_the_software.html to test the preprocessing on the example dataset. Is this helpful?

The plane of image acquisition should not prevent conducting preprocessing, as long as the XYZ do correspond to the right-left/anterior-posterior/superior-inferior directions (RAS convention), although I have only tested the software with coronal acquisitions so far. However, if you intend to apply slice-timing correction, you do have to make sure you are applying it over the right plane and direction.

Let me know if you need further support,
Gabriel DG

**Comment by nmskhan** [comment]
Thanks Gabriel for the suggestions.
I do see the example data sets and I wanted to run this on the Mac system. Let me check this out. 
Thanks
Ned
